AphroCiteTest.c

```diff
@@ -56,11 +56,37 @@ void testString(){
     Assert_True("Compare Sring Macro", CString_Compare(formattedString, copyString) == 0);
 };
 
+void TestUnitTestAssertTrueSuccess(T_UnitTest_TestResult* testResult) {
+    UnitTest_Assert_True(testResult, TRUE);
+
+    Console_Print_CString_Line("Should be reached!");
+}
+
+void TestUnitTestAssertTrueFail(T_UnitTest_TestResult* testResult) {
+    UnitTest_Assert_True(testResult, FALSE);
+    Console_Print_CString_Line("Should not be reached!");
+}
+
 // Main
 int main() {
     testConstants();
     testOperators();
     testMacros();
 
-    return EXIT_SUCCESS;
+    Console_Print_NextLine();
+    Console_Print_CString_Line(".---------------------.");
+    Console_Print_CString_Line("| New Unit Test Style |");
+    Console_Print_CString_Line(".---------------------.");
+    Console_Print_NextLine();
+
+    T_UnitTest_TestSuite testSuite = {0};
+    testSuite.name = "New Unit Test";
+    testSuite.silentMode = FALSE;
+
+    UnitTest_RunSingle(&testSuite, &TestUnitTestAssertTrueSuccess, "Test Success TRUE assert Unit Test");
+    UnitTest_RunSingle(&testSuite, &TestUnitTestAssertTrueFail, "Test Fail TRUE assert Unit Test");
+
+    UnitTest_PrintTestSuiteState(&testSuite);
+
+    return UnitTest_CreateExitCode(&testSuite);
 }
```

Comment:
no actually its two different things. print the test state and create the exit code. its just more explicit in my opinion because and it gives the end user more freedom